app/managers.py

```diff
@@ -0,0 +1,47 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema")
```

Comment:
Where do you close connection ?

Reply:
automatically? by analogy with the context manager?